### DISCUSSION TITLE
Can I clone SymbolicRegression.jl manually while running pysr.install()?

### DISCUSSION THREAD

**Discussion by WhiteGL** [MilesCranmer/PySR#294] [Q&A]
I always met difficulties about cloning git repos while running pysr.install(). But i can clone those repos manually. So can i put these repos like SymbolicRegression.jl somewhere so that i can skip the step of cloning while runing install()?

**Comment by MilesCranmer** [comment]
Yes, the `julia_project` parameter is for this purpose. Point it to the location of your local git repo.

https://astroautomata.com/PySR/api/#environment

To install dependencies manually, go to the SymbolicRegression.jl folder and type:

```bash
julia --project=. -e 'using Pkg; Pkg.instantiate()'
```

This is the equivalent of the `install()` step but run on your local folder
